template/config/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     index: path.resolve(__dirname, '../dist/index.html'),
     assetsRoot: path.resolve(__dirname, '../dist'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: './',
```

Comment:
why this change?

Reply:
This change allows the production app running in any localhost/{PATH}, instead of localhost/ only